services/app-api/utils/validation/validation.ts

```diff
@@ -2,6 +2,22 @@ import * as yup from "yup";
 import { nested, endDate, schemaMap } from "./schemaMap";
 import { AnyObject } from "../types/types";
 import { error } from "../constants/constants";
+import { APIGatewayEvent } from "aws-lambda";
+
+// validate report keys
```

Comment:
Suggestion: Since this isn't actually related to validation, we should move it to another utils file. I would either put it in utils under `dynamo` or make a new dir `handlers` and put it in a file under there.